settings.gradle

```diff
@@ -1,3 +1,3 @@
 rootProject.name = 'terra-cli'
 
-gradle.ext.cliVersion = '0.354.0'
+gradle.ext.cliVersion = '0.353.0'
```

Comment:
tag https://github.com/DataBiosphere/terra-cli/releases/tag/0.354.0 needs to be deleted too. Otherwise the release flow will again fail